# Research

## UniRef90

* [UniRef](https://www.uniprot.org/help/uniref)
* [UniRef90_P99999](https://www.uniprot.org/uniref/UniRef90_P99999)

## K-mer

### Theory

* [Definition of K-mer](https://en.wikipedia.org/wiki/K-mer)
* [k-mer Counting: Introduction](https://bioinfologics.github.io/post/2018/09/17/k-mer-counting-part-i-introduction/)
* [Alignment-free sequence comparison: benefits, applications, and tools](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1319-7)

### Bining

* [minimizers](https://homolog.us/blogs/bioinfo/2017/10/25/intro-minimizer/)
* [Low-memory k-mer analysis of metagenomic sequence data](https://towardsdatascience.com/optimizing-genomics-data-processing-on-apache-spark-3333d6d41ed6)
  * [discount](https://github.com/jtnystrom/discount)

### Hadoop

* [BioPig](https://github.com/JGI-Bioinformatics/biopig)
* [KCH - K-mer Counting on Hadoop](http://www.di-srv.unisa.it/KCH/)

### Spark

* [Adam](http://ampcamp.berkeley.edu/5/exercises/genome-analysis-with-adam.html)

* [Analyzing big datasets of genomic sequences: fast and scalable collection of *k*-mer statistics](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2694-8)
  * [FastKmer](https://github.com/maruscia/fastkmer)
* [discount](https://github.com/jtnystrom/discount)

